UnitTesting/T280/PhoneRepairShop_Code/PhoneRepairShop_Code.Tests/PhoneRepairShop_Code.Tests.csproj

```diff
@@ -35,9 +35,8 @@
     <WarningLevel>4</WarningLevel>
   </PropertyGroup>
   <ItemGroup>
-    <Reference Include="Autofac, Version=6.5.0.0, Culture=neutral, PublicKeyToken=17863af14b0044da, processorArchitecture=MSIL">
-      <SpecificVersion>False</SpecificVersion>
-      <HintPath>..\..\..\..\Bin\Autofac.dll</HintPath>
+    <Reference Include="Autofac, Version=7.0.0.0, Culture=neutral, PublicKeyToken=17863af14b0044da, processorArchitecture=MSIL">
+      <HintPath>..\packages\Autofac.7.0.0\lib\netstandard2.0\Autofac.dll</HintPath>
```

Comment:
> @MehranQadri , don't we need to use Autofac library from the Bin folder?

Not necessarily. You can install it using Nuget package manager as I have done in this instance. Both options are valid and are mentioned in the text:
![image](https://user-images.githubusercontent.com/123120313/232832487-02233f46-4754-414c-a118-36a574bb4de6.png)

Reply:
Could you please confirm that both options are OK with an sme? Because in unit tests we probably use Autofac for dependency injection, and for dependency injection we have the recommendation to use the version from the Bin folder. (see https://help.acumatica.com/(W(149))/Help?ScreenId=ShowWiki&pageid=67164f06-37ff-40c1-9f33-55e6aa43b44e)

Reply:
> Could you please confirm that both options are OK with an sme? Because in unit tests we probably use Autofac for dependency injection, and for dependency injection we have the recommendation to use the version from the Bin folder. (see https://help.acumatica.com/(W(149))/Help?ScreenId=ShowWiki&pageid=67164f06-37ff-40c1-9f33-55e6aa43b44e)

Thanks for sharing the article. Its best to stick with the Autofac dll provided in the acumatica instance. I will revert this change and remove the addition of the autofac package.